sysutils/AutoRaise/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sbmpost AutoRaise 1.5 v
+
+categories          sysutils
+platforms           macosx
+
+license             {GPL-2 GPL-3}
+maintainers         {@lhaeger gmx.net:lothar.haeger} openmaintainer
+
+description         AutoRaise raises application windows to the front and gives them focus on mouse hover
+long_description    ${description} after a configurable delay. There is also an option to warp the mouse \
+                    to the center of the activated window, using the cmd-tab key combination for example.
+
+homepage            htps://github.com/sbmpost/AutoRaise
```

Comment:
You misspelled https, but also the github portgroup already sets this correctly for you.
```suggestion
```

Reply:
Yep, double mistake: a typo and redundancy after I changed the source back to upstream instead of my fork. Thank!